### DISCUSSION TITLE
how to set a new file type  to search?

### DISCUSSION THREAD

**Discussion by yylays** [BurntSushi/ripgrep#1617] [General] [answered]
you can define a new type in Ackrc like "--type-set=inf:ext:inf", so i can search all the .inf file, however in rg option there is only "--type-add" ,i wander if you can define a file type to search. Thanks.

**Comment by BurntSushi** [comment, marked as answer]
Yes. It's documented in the guide: https://github.com/BurntSushi/ripgrep/blob/master/GUIDE.md#manual-filtering-file-types

In particular, you probably want to use `--type-add` in your configuration file: https://github.com/BurntSushi/ripgrep/blob/master/GUIDE.md#configuration-file

**Reply by yylays** [reply]
Thanks, i should read the document carefully. It helps.
